src/preferences/dialog/dlgprefreplaygain.cpp

```diff
@@ -3,24 +3,37 @@
 #include "controlobject.h"
 #include "util/math.h"
 
-#define kConfigKey "[ReplayGain]"
-
-static const int kReplayGainReferenceLUFS = -18;
-
-
-DlgPrefReplayGain::DlgPrefReplayGain(QWidget * parent, UserSettingsPointer  _config)
+namespace {
+const char* kConfigKey = "[ReplayGain]";
```

Comment:
While we're still using strings I'd prefer them to be shared rather than duplicated.